apps/angular/crud/src/app/app.component.ts

```diff
@@ -46,28 +51,16 @@ import { TaskComponent } from './ui/task/task.component';
   `,
   styles: [],
 })
-export class AppComponent implements OnInit, OnDestroy {
+export class AppComponent implements OnInit {
   private stodos = signal<Task[]>([]);
-  private sysSubs: Subscription = new Subscription();
   public get todos(): Task[] {
     return this.stodos();
   }
   public loaderService = inject(LoaderService);
   private store = inject(Store);
 
-  ngOnDestroy(): void {
-    this.sysSubs.unsubscribe();
-  }
-
   ngOnInit(): void {
     this.store.dispatch(taskActions.loadTasksAction());
-    this.sysSubs.add(
-      this.store.select(getTaskListSelector).subscribe({
-        next: (res) => this.stodos.set(res),
-        error: () => {
-          this.stodos.set([]);
-        },
-      }),
-    );
+    this.stodos = this.store.selectSignal(selectList) as WritableSignal<Task[]>;
```

Comment:
you don't have to initialize it inside ngoninit. You can do it immediatly  l.55

This is why signal are reactive, they will reactive to any changes